src/orca/services/nextflowtower/ops.py

```diff
@@ -41,3 +41,32 @@ def workspace(self) -> str:
             message = f"Config ({self.config}) does not specify a workspace."
             raise ConfigError(message)
         return self.config.workspace
+
+    def get_workflow(self, workflow_id: str) -> dict:
```

Comment:
A point of discussion.  Should `get_workflow` be a separate function, or... should we follow the [sevenbridges Ops class paradigm](https://github.com/Sage-Bionetworks-Workflows/py-orca/blob/56a40604f83c06d2cc07e503cc69033b832c8d2b/src/orca/services/sevenbridges/ops.py#L122).

Reply:
Makes sense to me. I'll make that change/